liana-gui/src/app/state/export.rs

```diff
@@ -0,0 +1,157 @@
+use std::{
+    path::PathBuf,
+    sync::{Arc, Mutex},
+};
+
+use iced::{Command, Subscription};
+use liana_ui::{component::modal::Modal, widget::Element};
+use tokio::task::JoinHandle;
+
+use crate::app::{
+    message::Message,
+    view::{self, export::export_modal},
+};
+use crate::daemon::Daemon;
+use crate::export::{self, get_path, ExportProgress};
+
+#[derive(Debug, Clone)]
+pub enum ExportMessage {
```

Comment:
Could `ExportMessage`, `ExportState` and perhaps some other parts of the code here also be moved to `liana-gui/src/export.rs`? They seem quite general and could also be reused in the installer, for example.